modules/tinymce/changelog.txt

```diff
@@ -1,3 +1,5 @@
+Version 5.3.1 (TBD)
+    Fixed the autocompleter throwing an exception when tooltipping was configured #TINY-6018
```

Comment:
this maybe makes more sense as "tooltips" than "tooltipping" to customers?

Reply:
It's not actually "tooltips" though, it's more of a popover that's currently undocumented and only used by mentions.

Reply:
that's not confusingly named at all XD I guess if we keep it as tooltipping, we could just add speech marks to show that it's a term? I can't think of any good alternative words.

Reply:
Dropping this entry since the feature isn't documented as discussed on slack.